pkg/operator/ceph/cluster/mon/mon.go

```diff
@@ -54,6 +54,8 @@ import (
 )
 
 const (
+	// EndpointName is the name of the endpoints with mon addresses
+	EndpointsName = "rook-ceph-active-mons"
```

Comment:
nit: since the const is not referenced outside this go package, we can make it private with lower case
```suggestion
	endpointsName = "rook-ceph-active-mons"
```